### DISCUSSION TITLE
Adding New Features

### DISCUSSION THREAD

**Discussion by hueychen27** [hueychen27/Popcorn_Clicker#1] [Ideas]
So people, we are going to have a discussion about adding in new features. Discuss below to see what you want to add.

**Comment by PANDASARECOOOL** [comment]
Maybe you should add Tourists Traps or something like that (it's just a tourist trap that forces tourists to pick the corn and "learn the rural experience" or whatever the heck tourist traps say) or maybe you could add some machinery, like trucks or ATVs that'll make the farm upgrade more efficient

**Reply by hueychen27** [reply]
So the what is the reward for the tourist traps?

**Reply by PANDASARECOOOL** [reply]
I guess a faster production or something? Maybe you could also make it more efficient by building airports near it? IDK, my ideas are trash

**Reply by hueychen27** [reply]
So you mean that the popcorn per second could increase depending on the amount of tourists. Like 15 tourists is a 15% boost for 1 minute. And don't say your ideas are trash.

**Reply by PANDASARECOOOL** [reply]
Yeah I guess that's what I meant, and maybe there'd be some randomness involved, like instead of buying a set amount of tourists, you buy a billboard or whatever, so then there would be a random amount of tourists that would come per minute

**Reply by hueychen27** [reply]
I think replacing tourists with popcorn investors should be better. Do you agree?

**Reply by PANDASARECOOOL** [reply]
Yeah I guess, and as I said on scratch, building entire airports and billboards to atract tourists that your multimillion company relies on is probably not a good decision

**Reply by hueychen27** [reply]
Remember that this is not real life. You can do whatever you want 👍🏼

**Reply by PANDASARECOOOL** [reply]
Good point, but even for a fictional game it sounds weird. Still though, maybe we could add both? Or maybe you could only choose one, like you'd either build airports and billboards to attract tourists or do whatever you do to attract investors

**Reply by hueychen27** [reply]
'k. I'll keep your idea in mind after we fix the upgrades bug.
